Assets/Scripts/WanderAI.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+using Utils;
+
+public class WanderAI
+{
+    // maxDistance for mapping random points onto the NavMesh
+    private float maxNavMeshMappingDistance;
+    // the number of tries to map a random point onto the NavMesh
+    private int mappingIterations;
+
+    public WanderAI ()
+    {
+        // init constants
+        maxNavMeshMappingDistance = 1.0f;
+        mappingIterations = 500; // just a high value to ensure random point generation
```

Comment:
Can you integrate those ones in the Constructor? Then we can, if needed play around with those values

Reply:
Or make them Consts

Reply:
Ja das ist gut. Consts reicht auch, ist noch bisschen kürzer haha